client/signup/steps/about/index.jsx

```diff
@@ -389,6 +389,21 @@ class AboutStep extends Component {
 					</span>
 				</FormLabel>
 
+				<FormLabel htmlFor="promote" className="about__checkbox-option">
+					<FormInputCheckbox
+						name="siteGoals"
+						id="promote"
+						onChange={ this.checkBoxHandleChange }
+						defaultChecked={ this.isCheckBoxChecked( 'promote_local_business' ) }
+						value="promote_local_business"
+						className="about__checkbox"
+						onKeyDown={ this.handleCheckboxKeyDown }
+					/>
+					<span className="about__checkbox-label">
+						{ translate( 'Promote your local business' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Promote your business or personal brand' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).